packages/lib-utils/src/k8s/hooks/k8s-watcher.ts

```diff
@@ -77,14 +77,14 @@ export const getReduxData = (immutableData, resource: WatchK8sResource) => {
 };
 
 export const getWatchData: GetWatchData = (resource, k8sModel, cluster) => {
-  if (!k8sModel || !resource || !resource.name || !resource.namespace) {
+  if (!k8sModel || !resource || !k8sModel.name || !resource.namespace) {
     return null;
   }
   const query = makeQuery(
     resource.namespace,
     resource.selector,
     resource.fieldSelector,
-    resource.name,
+    k8sModel.name,
```

Comment:
Hm.... wondering why this needed to be changed. We use the resource.name in console: https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/hooks/k8s-watcher.ts#L81